Can't resolve './locale' in '/home/circleci/project/node_modules/moment/src/lib/locale'

```
npm run build:dll
```
```
WARNING in ./node_modules/moment/src/lib/locale/locales.js
Module not found: Error: Can't resolve './locale' in '/home/circleci/project/node_modules/moment/src/lib/locale'
 @ ./node_modules/moment/src/lib/locale/locales.js 56:12-46
 @ ./node_modules/moment/src/lib/duration/constructor.js
 @ ./node_modules/moment/src/lib/duration/prototype.js
 @ ./node_modules/moment/src/lib/duration/duration.js
 @ ./node_modules/moment/src/moment.js
 @ dll reactBoilerplateDeps
```

--- a/internals/webpack/webpack.base.babel.js
+++ b/internals/webpack/webpack.base.babel.js
@@ -132,6 +132,9 @@ module.exports = options => ({
     modules: ['app', 'node_modules'],
     extensions: ['.js', '.jsx', '.react.js'],
     mainFields: ['browser', 'jsnext:main', 'main'],
+    alias: {
+      moment$: 'moment/moment.js',
+    },
   },
   devtool: options.devtool,
   target: 'web', // Make web variables accessible to webpack, e.g. window

